scipy/signal/_peak_finding.py

```diff
@@ -1188,7 +1188,8 @@ def filt_func(line):
 
 
 def find_peaks_cwt(vector, widths, wavelet=None, max_distances=None,
-                   gap_thresh=None, min_length=None, min_snr=1, noise_perc=10):
+                   gap_thresh=None, min_length=None, window_size=None,
```

Comment:
New keywords should be added at the end of the signature, to avoid breaking backwards compatibility (imagine a user specifies all inputs as positional args).